src/DaemonExec.ts

```diff
@@ -0,0 +1,138 @@
+import * as Docker from 'dockerode'
+
+interface ContainerModem {
+	demuxStream(
+		stream: NodeJS.ReadWriteStream,
+		stdoutStream: NodeJS.WritableStream,
+		stderrStream: NodeJS.WritableStream,
+	): void
+}
```

Comment:
Avoid repeating this interface definition both here and in the demo script; can you merge them and move it to a separate file (e.g. `DockerodeTypes.ts`)? Or, even better, make a PR to `@types/dockerode` 😉.

Reply:
The interfaces are different object types. Making a type definition file for all the modems would be nice, though it might take a while, and probably isn't necessary for our project.

Reply:
Oh yeah, my bad, somehow I didn't realize they have different names 😅, resolving this